src/wallet/wallet2.cpp

```diff
@@ -862,7 +862,12 @@ bool wallet2::get_multisig_seed(std::string& seed, const epee::wipeable_string &
 
   if (raw)
   {
-    seed = epee::string_tools::buff_to_hex_nodelimer(data);
+    for (size_t i = 0; i < data.size(); ++i)
+    {
+      const int c = data.data()[i];
+      seed += (c >> 4) & 0xf;
+      seed += c & 0xf;
+    }
```

Comment:
Shouldn't this use the hex character table similar to epee/src/hex.cpp?
```c++
    template<typename T>
    void write_hex(T&& out, const span<const std::uint8_t> src)
    {
      static constexpr const char hex[] = u8"0123456789abcdef";
      static_assert(sizeof(hex) == 17, "bad string size");
      for (const std::uint8_t byte : src)
      {
        *out = hex[byte >> 4];
        ++out;
        *out = hex[byte & 0x0F];
        ++out;
      }
    }
```

Reply:
Indeed, fixing.